src/runtime/aarch64.rs

```diff
@@ -45,6 +45,23 @@ pub fn detect_features<T: linux::FeatureQuery>(mut x: T) -> usize {
     value
 }
 
+/// Probe the ELF Auxiliary vector for hardware capabilities
+///
+/// The values are part of the platform-specific `asm/hwcap.h`
```

Comment:
Could you maybe add a link to the linux header where hwcap is defined? That is, here: https://github.com/torvalds/linux/blob/master/arch/arm64/include/uapi/asm/hwcap.h

Reply:
Sure.

Reply:
Thanks!